doc-Infrastructure_Migration_Solution_Guide/cfme/doc/modules/con_Advanced_migration_option-_automating_pre-_and_post-migration_tasks_with_ansible.adoc

```diff
@@ -0,0 +1,6 @@
+[id="Advanced_migration_option-_automating_pre-_and_post-migration_tasks_with_ansible"]
+= Advanced Migration Option: Automating Pre- and Post-Migration Tasks with Ansible
+
+You can automate certain tasks before or after a virtual machine is migrated, for example, removing web server virtual machines from a load-balancing pool before migration and adding them back to the pool after migration. You create an Ansible service catalog item that contains the playbook and select it while creating the infrastructure migration plan.
+
+Red Hat recommends using the xref:Ims_rhel_pre-migration_ansible_playbook_example[IMS RHEL Pre-migration Ansible Playbook Example] to configure the target Red Hat Enterprise Linux virtual machines.
```

Comment:
CCS discourages use of "Red Hat recommends" and would like to see a more prescriptive approach taken in documentation. In this instance I would revise from "Red Hat recommends..." to "Use xref:<link> to configure Red Hat Enterprise Linux machines <for this reason>."